resources/views/organizations/index.blade.php

```diff
@@ -0,0 +1,42 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        <div>
+            <h1>Organizations</h1>
+            @auth
+                <a href="{{ route('organizations.create') }}">Create new organization</a>
+            @endauth
+            @if ($organizations->count())
+                @foreach ($organizations as $organization)
+                    <div>
+                        {{ $organization->name }}
+                        {{ $organization->description }}
+                        {{ $organization->location }}
+                        {{ $organization->organization_type }}
+                        {{ $organization->foundation_date }}
+                        {{ $organization->number_of_employers }}
+                        <a href="{{ $organization->website_url }}"> Website </a>
+                        <a href="{{ $organization->facebook_url }}"> Facebook </a>
+                        <a href="{{ $organization->linkedin_url }}"> Linkedin </a>
+                        <a href="{{ $organization->instagram_url }}"> Instagram </a>
+                        <a href="{{ $organization->youtube_url }}"> YouTube </a>
+                        <a href="{{ $organization->twitter_url }}"> Twitter </a>
+                        <a href="{{ $organization->github_url }}"> Github </a>
```

Comment:
This is a lot of informations for index page... I would prefer show that on show view.